packages/minimongo/minimongo.js

```diff
@@ -221,14 +221,26 @@ LocalCollection.Cursor.prototype.fetch = function () {
  * @locus Anywhere
  * @returns {Number}
  */
-LocalCollection.Cursor.prototype.count = function () {
+LocalCollection.Cursor.prototype.count = function (applySkipLimit) {
   var self = this;
+  // Have to respect `applySkipLimit`, so we remove limit & skip if `applySkipLimit`
```

Comment:
I believe so. `fetch` is just executed after `count` in this case. Where's the problem?